i18n/zh/docusaurus-plugin-content-docs/current/start/wasmedge/features.md

```diff
@@ -69,7 +69,11 @@ WasmEdge now supports:
 
 It is easy to build customized WasmEdge runtime with native host functions in C, Go, and Rust.
 
-Or you could build your own plug-ins for WasmEdge in C++, C, or Rust (WIP).
+Or you could build your own plug-ins for WasmEdge in,
+
+- [Rust](../../contribute/plugin/develop_plugin_rustsdk) - WIP
```

Comment:
Currently we have the plugin guide for Rust SDK. Its not WIP anymore.

Reply:
@alabulei1 Could you please lmk about this? Because @q82419 refactoring had put them in WIP tags

Reply:
We do have one. https://wasmedge.org/docs/contribute/plugin/develop_plugin_rustsdk

Reply:
Done ✅